testing/opensmtpd-extras/APKBUILD

```diff
@@ -0,0 +1,92 @@
+# Contributor: Shiz <hi@shiz.me>
+# Maintainer: Shiz <hi@shiz.me>
+pkgname=opensmtpd-extras
+pkgver=201703132115
+pkgrel=0
+pkgdesc="OpenSMTPD addons"
+url="https://opensmtpd.org/"
+arch="all"
+license="ISC"
+depends=""
+_depends_filter_monkey="libressl"
+_depends_queue_python="python2"
+_depends_scheduler_python="python2"
+_depends_table_mysql="mariadb-client-libs"
+_depends_table_postgres="libpq"
+_depends_table_python="python2"
+_depends_table_redis="hiredis"
+_depends_table_sqlite="sqlite-libs"
```

Comment:
According to the [build log](https://travis-ci.org/alpinelinux/aports/builds/218695592#L1268) all these dependencies are auto-detected via dynamic links, so you can remove all these `_depends` and so simplify the abuild even more. :)

```
>>> opensmtpd-filter-monkey*: Tracing dependencies...
    libressl
    so:libc.musl-x86_64.so.1
    so:libcrypto.so.38
    so:libevent-2.1.so.6

>>> opensmtpd-queue-python*: Tracing dependencies...
    python2
    so:libc.musl-x86_64.so.1
    so:libcrypto.so.38
    so:libpython2.7.so.1.0

>>> opensmtpd-scheduler-python*: Tracing dependencies...
    python2
    so:libc.musl-x86_64.so.1
    so:libpython2.7.so.1.0

>>> opensmtpd-table-mysql*: Tracing dependencies...
    mariadb-client-libs
    so:libc.musl-x86_64.so.1
    so:libmysqlclient.so.18

>>> opensmtpd-table-postgres*: Tracing dependencies...
    libpq
    so:libc.musl-x86_64.so.1
    so:libpq.so.5

>>> opensmtpd-table-python*: Tracing dependencies...
    python2
    so:libc.musl-x86_64.so.1
    so:libpython2.7.so.1.0

>>> opensmtpd-table-redis*: Tracing dependencies...
    hiredis
    so:libc.musl-x86_64.so.1
    so:libhiredis.so.0.13

>>> opensmtpd-table-sqlite*: Tracing dependencies...
    sqlite-libs
    so:libc.musl-x86_64.so.1
    so:libsqlite3.so.0
```